packages/opentelemetry-core/package.json

```diff
@@ -63,6 +64,7 @@
     "webpack": "^4.35.2"
   },
   "dependencies": {
-    "@opentelemetry/types": "^0.0.1"
+    "@opentelemetry/types": "^0.0.1",
+    "console-log-level": "^1.4.1"
```

Comment:
Is this needed? I would generally prefer taking dependencies that have a larger value-add and if it's simple to just inline it in the library (less risk of security issues, etc.)